examples/mnist/train_mnist_custom_loop.py

```diff
@@ -88,18 +86,14 @@ def main():
                 sum_accuracy = 0
                 sum_loss = 0
                 # Enable evaluation mode.
-                with configuration.using_config('train', False):
-                    # This is optional but can reduce computational overhead.
-                    with chainer.using_config('enable_backprop', False):
-                        for batch in test_iter:
-                            x, t = convert.concat_examples(batch,
-                                                           args.gpu)
-                            x = chainer.Variable(x)
-                            t = chainer.Variable(t)
-                            loss = model(x, t)
-                            sum_loss += float(loss.data) * len(t.data)
-                            sum_accuracy += (float(model.accuracy.data) *
-                                             len(t.data))
+                # This is optional but can reduce computational overhead.
```

Comment:
I think it's better to say that only `enable_backprop` is optional. (setting `train` to `False` is not optional)

Reply:
Thanks! I will fix